src/frontend/komponenter/Fagsak/Behandlingsresultat/EøsPeriode/FamilieLandvelger.tsx

```diff
@@ -4,6 +4,13 @@ import classNames from 'classnames';
 import styled from 'styled-components';
 
 import { Label } from '@navikt/ds-react';
+import {
+    ABorderDanger,
+    ABorderStrong,
+    AGray100,
+    AGray300,
+    ATextDanger,
+} from '@navikt/ds-tokens/dist/tokens';
```

Comment:
Se migreringsguide for dokumentasjon av mapping fra gamle til nye tokens https://aksel.nav.no/grunnleggende/kode/migrering#h018af102ad07